docs/settings/security-settings.asciidoc

```diff
@@ -30,6 +30,112 @@ You do not need to configure any additional settings to use the
 
 |===
 
+[float]
```

Comment:
__issue:__ same comment as in 7.5 version of this PR https://github.com/elastic/kibana/pull/96478#discussion_r609593343. Whatever we do for 7.5 can be fully replicated for 7.6 as well.